tests/nominal/test_NominalToIntegerTransformer.py

```diff
@@ -1,61 +1,37 @@
 import pandas as pd
 import pytest
 import test_aide as ta
+from test_BaseNominalTransformer import GenericBaseNominalTransformerTests
 
 import tests.test_data as d
-import tubular
+from tests.base_tests import (
+    ColumnStrListInitTests,
+    GenericFitTests,
```

Comment:
Same Q as on the other PR, should these generic classes be imported from test_BaseNominalTransformer?

Reply:
Have made same changes here as I made in  #259  https://github.com/lvgig/tubular/pull/261/commits/c896b06f09f8804fd8c8c3e69d7b1716964bf8be